src/components/input/base/index.tsx

```diff
@@ -73,33 +75,9 @@ const Input = forwardRef<HTMLInputElement, InputProps>(
     },
     ref,
   ) => {
-    const wrapperRef = useRef<HTMLDivElement>(null)
-    useEffect(() => {
-      if (disabled) {
-        wrapperRef.current?.setAttribute('data-disabled', 'disabled')
-      } else {
-        wrapperRef.current?.removeAttribute('data-disabled')
-      }
-    }, [disabled])
-    const clearHover = (event: React.FocusEvent<HTMLInputElement>) => {
-      if (event.currentTarget?.parentElement?.getAttribute('data-hovered')) {
-        event.currentTarget?.parentElement?.removeAttribute('data-hovered')
-      }
-    }
-    const handleFocus = (event: React.FocusEvent<HTMLInputElement>) => {
-      clearHover(event)
-      event.target.parentElement?.setAttribute('data-focused', 'focused')
-      onFocus?.(event)
-    }
-    const handleBlur = (event: React.FocusEvent<HTMLInputElement>) => {
-      clearHover(event)
-      event.target.parentElement?.removeAttribute('data-focused')
-      onBlur?.(event)
-    }
     return (
       <div className="flex flex-col">
         <div
-          ref={wrapperRef}
           className={qtMerge(
```

Comment:
`qtJoin` would be better here

Reply:
Basically, use `qtJoin` instead of `qtMerge` wherever you are not directly passing the className from props.

Reply:
@hasan-deriv, I wasn't aware of this, so thank you for sharing. But in this case, I'm passing the `className` from props. as seen in the screenshot below
![Uploading className.png…]()